dependencies/README.md

```diff
@@ -0,0 +1,62 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that inspects classpaths for [static linkage check](
```

Comment:
"Static Linkage Checker is a tool performs a [static linkage check](...) on a classpath."